wasp/src/main/java/com/orhanobut/wasp/VolleyNetworkStack.java

```diff
@@ -187,8 +198,13 @@ public VolleyRequest(int method, String url, RequestCreator request, VolleyListe
     }
 
     @Override
-    protected void deliverResponse(T response) {
-      listener.onResponse(response);
+    protected void deliverResponse(Response response) {
```

Comment:
when we use the new signature, we can remove all response operations but leave the logging.

```java
protected void deliverResponse(Response response) {
   response.log();
}
```